modules/runtime/nuxeo-runtime-migration/src/main/java/org/nuxeo/runtime/migration/MigrationServiceImpl.java

```diff
@@ -470,8 +471,7 @@ public void runStep(String id, String step) {
                 log.error("Exception during execution of step: {} for migration: {}", step, id, t);
             }
             // after the migrator is finished, change state, except if shutdown is requested or exception
-            String state = t != null || migrationContext.isShutdownRequested()
-                    ? stepDescr.fromState
+            String state = t != null || migrationContext.isShutdownRequested() ? stepDescr.fromState
```

Comment:
didn't feel worth a separate commit. Unless ?...

Reply:
It's OK to me, no need for a cleanup commit.